ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala

```diff
@@ -91,6 +111,148 @@ class IngestsTrackerApiFeatureTest
     }
   }
 
+  describe("PATCH /ingest/:id") {
+    val ingest = createIngestWith(
+      createdDate = Instant.now(),
+      events = Seq(createIngestEvent, createIngestEvent).sortBy {
+        _.createdDate
+      },
```

Comment:
Any reason to sort these?

Reply:
This is copy/pasted from the ingests tests - I presume this is intended to create realistically date ordered events. I don't think we rely on the order in these tests.